packages/components/src/components/popover/popover-utils.ts

```diff
@@ -160,3 +172,20 @@ export const onDocumentKeydown = (e: KeyboardEvent): void => {
     }
   }
 };
+
+export const addNativeScrollListeners = (host: HTMLElement, table: HTMLElement, nativePopover: HTMLElement): void => {
+  const tableScrollArea = table.shadowRoot
+    .querySelector(getPrefixedTagNames(host).pScroller)
+    .shadowRoot.querySelector('.scroll-area');
```

Comment:
Risky.. and could break anytime if class changes? Can we select it differently or sync the class name somehow?